os/arch/arm/src/armv7-m/up_lazyexception.S

```diff
@@ -371,7 +375,7 @@ exception_common:
 #ifdef CONFIG_ARMV7M_USEBASEPRI
 	msr		basepri, r3				/* Restore interrupts priority masking */
 #ifndef CONFIG_ARCH_HIPRI_INTERRUPT
-	cpsie	i						/* Re-enable interrupts */
+	/*cpsie   i*/						/* Re-enable interrupts */
```

Comment:
same as above

Reply:
modified the code to make it macro protected